src/Components/CreateImageWizard/steps/googleCloud.js

```diff
@@ -19,6 +19,7 @@ const PopoverInfo = ({ appendTo }) => {
         hasAutoWidth
         maxWidth='35rem'
         headerContent={ 'Valid account types' }
+        flipBehavior={['right', 'down', 'up', 'left']}
```

Comment:
down should be bottom and up should be top

Reply:
flipBehaviour can have the following options: 

'flip' \| (     \| 'top'     \| 'bottom'     \| 'left'     \| 'right'     \| 'top-start'     \| 'top-end'     \| 'bottom-start'     \| 'bottom-end'     \| 'left-start'     \| 'left-end'     \| 'right-start'     \| 'right-end'   )

Reply:
> down should be bottom and up should be top
Smh thanks. Was looking at that too long and obviously didn't see it

Reply:
haha I know that feeling